### DISCUSSION TITLE
Welcome to GirlScript-Asansol Discussions!

### DISCUSSION THREAD

**Discussion by regdocs** [gsasansol/GirlScript-Asansol#98] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DeleMike** [comment]
Hi @girlscript-asansol , I am Akindele Michael and I hope to be involved in the building of the web application. I have experience with HTML/CSS/JS and Express. How can I contribute to this project? I went through all the issues and they are already assigned to someone. The Discord server seems to lag from my end too

**Reply by regdocs** [reply]
Hello and welcome Akindele!

Try logging into Discord on Chrome/Firefox desktop. The server lag is more like an Android issue afaik. The Discord server is the standard way we're accepting contributions on our repository right now. However, you may try creating new issues on GitHub as well since we're beginning to allow external contributors to claim issues raised directly on GitHub.

Also, since you informed us of the technical problems on your end, you're welcome to raise your issues on GitHub itself. Take a look at our progress so far and start a thread to begin contributing.

Happy Hacktoberfest!

**Reply by regdocs** [reply]
@DeleMike Can you try opening the gsa server again? We've been receiving similar prompts from other users and it was high time we tried to resolve the lag.

**Reply by DeleMike** [reply]
@jay-io At the moment, I can confirm that it is working smoothly. Thanks!
